vizro-core/src/vizro/actions/_filter_interaction.py

```diff
@@ -51,7 +51,8 @@ def pre_build(self):
         figure_ids_on_page = [
             model.id
             for model in cast(
-                Iterable[FigureType], model_manager._get_models(FIGURE_MODELS, page=model_manager._get_model_page(self))
+                Iterable[FigureType],
```

Comment:
Rough comment to myself: look at action TODOs and figure out how to make this work inside container.